util/chunk/chunk.go

```diff
@@ -304,16 +305,28 @@ func (c *Chunk) PreAlloc(row Row) {
 			continue
 		}
 		// Grow the capacity according to golang.growslice.
+		// Implementation differences with golang:
+		// 1. We double the capacity when `dstCol.data < 1024*elemLen bytes` but
+		// not `1024 bytes`.
+		// 2. We expand the capacity to 1.5*originCap rather than 1.25*originCap
+		// during the slow-increasing phase.
```

Comment:
Why do we make this change?

Reply:
The length of the `chunk.data` slice will increase rapidly, to reduce the frequency of memory allocation, I did this change.